<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="registratePushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>270</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>401</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="regUsernameLabel">
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="regUserNameLineEdit">
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="regPasswordLabel">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="regPasswordLineEdit">
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="regEMailLabel">
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="regEMailLineEdit">
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelManufacturer">
      <property name="text">
       <string>Manufacturer:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditManufacturer">
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelType">
      <property name="text">
       <string>Type:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEditType">
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labelModel">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEditModel">
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelDescription">
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTextEdit" name="textEditDescription">
      <property name="font">
       <font>
        <family>Bitstream Charter</family>
        <pointsize>14</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>120</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(99, 99, 99);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>No image</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Load image</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInfoToUser">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>681</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Label info to user</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="graphics.qrc">:/new/prefix1/Graphics/sf_2.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>registratePushButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>labelImage</zorder>
  <zorder>buttonImage</zorder>
  <zorder>labelInfoToUser</zorder>
 </widget>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
